src/components/CropSelector/CropDetailsModal.js

```diff
@@ -36,9 +38,9 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const CropDetailsModalComponent = (props) => {
+function CropDetailsModalComponent(props) {
   const { state } = useContext(Context);
-  let crop = props.crop;
+  const { crop, setModalOpen, modalOpen } = props;
```

Comment:
did the linter do this? can we make it put props in the arg?

Reply:
nope it just didnt catch files that didnt have props. issues so some of these still lingered glad you caught them